## 95% of end-end tests have passed
```
Total executed tests: 55
Failed tests: 0
Expected to fail tests: 3
Passed tests: 52
Not executed tests: 1
```
```
IDs of not executed tests: 741054 
```
```
IDs of expected to fail tests: 702948,702859,703629 
```
<h3>Not executed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=tests/remove-xfail&PR_ID=21681&APK_NAME=21681.apk&TR_CASE_IDS=741054">Rerun not executed tests</a></li></details><h3>Expected to fail tests (3)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738878">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `BaseElement` by `accessibility id`: `toast-content` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:481: in test_community_one_image_send_reply
    toast_element_text = toast_element.text
../views/base_element.py:237: in text
    return self.find_element().text
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:90: in text
    return self._execute(Command.GET_ELEMENT_TEXT)["value"]
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.accessibilityId: toast-content' does not exist in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.accessibilityId: toast-content' does not exist in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:117)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.GetText.safeHandle(GetText.java:20)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:259)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:253)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:77)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012) 
```

<b>[[Skipped due to waku issue on staging fleet]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#266">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#236">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738882">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1127: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Skipped due to waku issue on staging fleet]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a853feba304e16838fe6436d583524?auth=134dae937215677762461c1bea2d92a0#317">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b13794724544e04be6d8bf9c819c02f?auth=ceb9be210875473a322a52611eede304#301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738895">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1213: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:192: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Skipped due to waku issue on staging fleet]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a853feba304e16838fe6436d583524?auth=134dae937215677762461c1bea2d92a0#512">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b13794724544e04be6d8bf9c819c02f?auth=ceb9be210875473a322a52611eede304#474">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (52)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738869">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#298">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#249">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738880">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#423">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#363">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738885">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#551">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#436">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738872">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#325">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738870">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#392">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#332">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738868">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#212">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#137">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738867">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#167">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738897">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#588">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#463">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738871">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#186">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738887">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/716c0da4089541438f86a216b1a173c9?auth=163d3266e64b69bd85c4549c1f40f6cc#240">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/737d6cbc018c4320858cc1ff04983137?auth=1389882fb59f7a9974379b3f20184e68#171">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738893">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f903a65c3a9d4021b5c9f4ec4f205fee?auth=25ba51edf09bcb5f7791e37976fb2941#118">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/781e05011e8c4e259a20cbb70d2d93a7?auth=1aafcaf9dca3bd6db40f3cc73815c370#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738860">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f903a65c3a9d4021b5c9f4ec4f205fee?auth=25ba51edf09bcb5f7791e37976fb2941#107">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/781e05011e8c4e259a20cbb70d2d93a7?auth=1aafcaf9dca3bd6db40f3cc73815c370#94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738861">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f903a65c3a9d4021b5c9f4ec4f205fee?auth=25ba51edf09bcb5f7791e37976fb2941#187">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/781e05011e8c4e259a20cbb70d2d93a7?auth=1aafcaf9dca3bd6db40f3cc73815c370#145">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738873">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a853feba304e16838fe6436d583524?auth=134dae937215677762461c1bea2d92a0#313">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b13794724544e04be6d8bf9c819c02f?auth=ceb9be210875473a322a52611eede304#277">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738862">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a853feba304e16838fe6436d583524?auth=134dae937215677762461c1bea2d92a0#148">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b13794724544e04be6d8bf9c819c02f?auth=ceb9be210875473a322a52611eede304#119">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738865">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8a853feba304e16838fe6436d583524?auth=134dae937215677762461c1bea2d92a0#196">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b13794724544e04be6d8bf9c819c02f?auth=ceb9be210875473a322a52611eede304#174">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738899">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738898">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738877">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#482">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#311">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738851">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#103">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#94">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738856">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#373">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#228">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738852">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#228">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#181">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738854">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#577">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#418">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738866">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#432">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#279">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738859">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738891">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4a3014644eff4e06907fe4f5c9678d2e?auth=2e7c69fbf0022cf7c9922b78f42962cb#515">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9c98b076a743e3bb03c74c8f2d77b8?auth=27f34157ebaad5fe9d7804b014d28e45#357">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738884">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a04c142b391e4d00b16dd5ca6ce3d0ec?auth=b07a1748cd73105ec4acf7b907da15d1#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eb39892ef3af46fdad104af48f843668?auth=503c849993c1b0b5594710324c936cd7#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738883">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a04c142b391e4d00b16dd5ca6ce3d0ec?auth=b07a1748cd73105ec4acf7b907da15d1#122">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eb39892ef3af46fdad104af48f843668?auth=503c849993c1b0b5594710324c936cd7#95">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738896">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53396c2eabf349f1b93aef20c9d3c4b3?auth=c765491a04871839935149f2c410a4bf#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738857">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53396c2eabf349f1b93aef20c9d3c4b3?auth=c765491a04871839935149f2c410a4bf#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738901">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/53396c2eabf349f1b93aef20c9d3c4b3?auth=c765491a04871839935149f2c410a4bf#189">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738876">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fdde98c129eb47a49963e289e88d4a36?auth=9e42e34aeb4d1614aaa78fe91f162f80#110">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca79f70230544309a4e533307a1cf4b1?auth=5d78f35a5d2741ef53b3222b5efe5ff5#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738875">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fdde98c129eb47a49963e289e88d4a36?auth=9e42e34aeb4d1614aaa78fe91f162f80#66">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca79f70230544309a4e533307a1cf4b1?auth=5d78f35a5d2741ef53b3222b5efe5ff5#50">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738858">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fdde98c129eb47a49963e289e88d4a36?auth=9e42e34aeb4d1614aaa78fe91f162f80#146">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ca79f70230544309a4e533307a1cf4b1?auth=5d78f35a5d2741ef53b3222b5efe5ff5#197">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738888">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#117">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#97">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738863">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#166">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#88">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738864">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#339">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#219">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738853">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#375">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#247">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738889">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#298">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#183">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#154">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738892">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4270068c5c3b4cbc9a10c9a5180c17d9?auth=13ae365feede5c48cf60a313bedd3312#503">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c4c245ac1f0746ad9fe94b638945d95a?auth=9ea1384422f4d3405c76823191388c4c#292">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8d224109b1ac4032873b607d713ec481?auth=e404bfcdb3ed782fa5e954b3d6c8e8bc#258">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738902">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72f63aee3e614dd2af378af65fea411a?auth=a68a33fa1b92e6d6d708b0667c41f7de#159">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e84b91cbe4574b3587ff67aa677ffbc8?auth=372e0a7ba3deb7746b7b2d456a714f7c#4">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3fa5a5e1bcb943ff9b8af3dc13b9a7ae?auth=ac372bf8e8aeeb9d871564646eda974e#60">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738903">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72f63aee3e614dd2af378af65fea411a?auth=a68a33fa1b92e6d6d708b0667c41f7de#163">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e84b91cbe4574b3587ff67aa677ffbc8?auth=372e0a7ba3deb7746b7b2d456a714f7c#30">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3fa5a5e1bcb943ff9b8af3dc13b9a7ae?auth=ac372bf8e8aeeb9d871564646eda974e#66">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738904">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/72f63aee3e614dd2af378af65fea411a?auth=a68a33fa1b92e6d6d708b0667c41f7de#230">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e84b91cbe4574b3587ff67aa677ffbc8?auth=372e0a7ba3deb7746b7b2d456a714f7c#124">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3fa5a5e1bcb943ff9b8af3dc13b9a7ae?auth=ac372bf8e8aeeb9d871564646eda974e#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738900">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738905">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738886">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#244">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738855">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738874">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#47">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738879">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738890">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738894">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d4e35aa2a4aa466a824a149120dfa5ca?auth=f594c05f83dba68ab242b76a1b212b98#353">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3738881">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b78abd18e9c44d84a136494c7a85a125?auth=00dcb6a9a68b27b1ca1a41cafac0aa49#126">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2ac680d794704679a92c1ebc318a4b7d?auth=67ad9a939201faf73e030294a9450824#93">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/test/appium/tests/activity_center/test_activity_center.py
+++ b/test/appium/tests/activity_center/test_activity_center.py
@@ -237,7 +237,6 @@ class TestActivityCenterContactRequestMultipleDevicePR(MultipleSharedDeviceTestC
 
 @pytest.mark.xdist_group(name="new_four_2")
 @marks.nightly
-@marks.xfail(run=False, reason="Skipped due to waku issue on staging fleet")
 class TestActivityMultipleDevicePR(MultipleSharedDeviceTestCase):
 
     def prepare_devices(self):
@@ -378,7 +377,6 @@ class TestActivityMultipleDevicePR(MultipleSharedDeviceTestCase):
 
 @pytest.mark.xdist_group(name="new_six_2")
 @marks.nightly
-@marks.xfail(run=False, reason="Skipped due to waku issue on staging fleet")
 class TestActivityMultipleDevicePRTwo(MultipleSharedDeviceTestCase):
 
     def prepare_devices(self):

--- a/test/appium/tests/critical/chats/test_public_chat_browsing.py
+++ b/test/appium/tests/critical/chats/test_public_chat_browsing.py
@@ -301,7 +301,6 @@ class TestCommunityOneDeviceMerged(MultipleSharedDeviceTestCase):
 
 @pytest.mark.xdist_group(name="new_three_2")
 @marks.nightly
-@marks.xfail(run=False, reason="Skipped due to waku issue on staging fleet")
 class TestCommunityMultipleDeviceMerged(MultipleSharedDeviceTestCase):
 
     def prepare_devices(self):
@@ -832,7 +831,6 @@ class TestCommunityMultipleDeviceMerged(MultipleSharedDeviceTestCase):
 
 @pytest.mark.xdist_group(name="new_five_2")
 @marks.nightly
-@marks.xfail(run=False, reason="Skipped due to waku issue on staging fleet")
 class TestCommunityMultipleDeviceMergedTwo(MultipleSharedDeviceTestCase):
 
     def prepare_devices(self):

--- a/test/appium/tests/critical/test_deep_and_universal_links.py
+++ b/test/appium/tests/critical/test_deep_and_universal_links.py
@@ -75,7 +75,6 @@ class TestDeepLinksOneDevice(MultipleSharedDeviceTestCase):
         self.errors.verify_no_errors()
 
     @marks.testrail_id(739307)
-    @marks.xfail(run=False, reason="Skipped due to waku issue on staging fleet")
     def test_deep_links_communities(self):
         closed_community_name, snt_community_name = "closed community", "SNT community"
         self.home.navigate_back_to_home_view()

